http-ballerina/src/http/http_commons.bal

```diff
@@ -252,9 +252,11 @@ isolated function uuid() returns string {
 # + method - http method of the request
 # + statusCode - status code of the response
 isolated function addObservabilityInformation(string path, string method, int statusCode, string url) {
+    string statusCodeConverted = statusCode.toString();
     error? err = observe:addTagToSpan(HTTP_URL, path);
     err = observe:addTagToSpan(HTTP_METHOD, method);
-    err = observe:addTagToSpan(HTTP_STATUS_CODE_GROUP, getStatusCodeRange(statusCode));
+    err = observe:addTagToMetrics(HTTP_STATUS_CODE_GROUP, getStatusCodeRange(statusCodeConverted));
+    err = observe:addTagToSpan(HTTP_STATUS_CODE, statusCodeConverted);
     err = observe:addTagToSpan(HTTP_BASE_URL, url);
```

Comment:
We need to add `HTTP_URL`, `HTTP_METHOD` & `HTTP_BASE_URL` as metrics tags too. Because it was previously available.